### ISSUE TITLE
Always get same location on the app

### ISSUE THREAD

**Issue by verdonarthur** [MediaComem/modos#64]
On the app, event after reinstalling, it's always the same initial position given at the first run of the app that is send to the server.

**Comment by verdonarthur** [comment]
Some observation given by participant:
```
Premier essai
J'ai désinstallé la version 0.1.2 et ai installé la version 0.1.1. Après redémarrage, j'ai du donner à nouveau les autorisations d'accès à la géolocalisation et à l'appareil photo.
Connexion avec modos@mail.com et observation depuis le parking de jumbo (id=261, sur la photo, un post-it avec les coordonnées de l'endroit). L'observation a de nouveau été positionnée au même endroit que celles d'hier.

Deuxième essai
Toujours avec v0.1.1. Dans les settings du téléphone -> Applications -> Modos -> Stockage : j'ai vidé le cache.
Re connexion avec modos@mail.com. Observation depuis ma terrasse (id=262, post-it avec GPS) -> toujours empilée par dessus les autres.

Troisième essai
v0.1.1, dans les settings -> effacer le cache ET effacer les données
Login avec modos@mail.com. ré-autorisation de l'accès à la localisation, etc. Observation depuis ma terrasse (id=263) toujours au même endroit.

Quatrième essai
dans les settings, effacer le cache et les données. Désinstallation de l'application. Réinstallation de la version 0.1.1.
Login avec modos@mail.com. ré-autorisation de l'accès à la localisation, observations depuis ma terrasse (id=264) toujours empilée sur le restaurant chinois de la gare...
```

**Comment by verdonarthur** [comment]
Fixed in #73
